rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/OperationServiceV2.java

```diff
@@ -296,6 +298,15 @@ private Operation generateExampleJson(final Class<? extends Operation> opClass)
         return operationFields;
     }
 
+    private String getOperationOutputType(final Operation operation) {
+        if (operation instanceof Output) {
+            final Type outputType = ((Output) operation).getOutputTypeReference().getType();
+            return outputType.getTypeName();
```

Comment:
Please can you format the output type name in the same way that is done for JsonSerialisationUtil.getSerialisedFieldClasses